script/version.sh

```diff
@@ -2,7 +2,7 @@
 set -e
 
 prefix=`cat VERSION` # Retrieve the version prefix
-timestamp=`date +%s` # Calculate the current timestamp
+timestamp=`date +%Y%m%d%H%M%S` # Calculate the current timestamp in ISO8601 format
```

Comment:
```suggestion
timestamp=`date +%Y%m%d%TH%M%S` # Calculate the current timestamp in ISO8601 format
```

I don't feel strongly about this, but I think the `T` is a standard separator for "date" and "time" in the ISO formats, and shouldn't break the sorting.

Reply:
According to Wikipedia, both are valid. (the full one has dashes) but T sounds fine, the `pre` segment is alpha-sorted